### PR TITLE
ci-operator: create logfile

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

### CODE REVIEW

File: cmd/ci-operator/main.go [hunk 1]
```diff
@@ -279,7 +279,7 @@ func setupLogger() (*secrets.DynamicCensor, io.Closer, error) {
 	if !set {
 		return &censor, nil, nil
 	}
-	verboseFile, err := os.Open(filepath.Join(artifactDir, "ci-operator.log"))
+	verboseFile, err := os.Create(filepath.Join(artifactDir, "ci-operator.log"))
```
Comment (alvaroaleman) [inline cmd/ci-operator/main.go:282]: What if it already exists?

Comment (stevekuznetsov) [reply cmd/ci-operator/main.go:282]: Well, it clearly doesn't :)

Comment (alvaroaleman) [reply cmd/ci-operator/main.go:282]: But how did it go through e2e then?

Comment (stevekuznetsov) [reply cmd/ci-operator/main.go:285]: So `Create` does not fail when the file exists, it just truncates it. I think this is fine behavior. Added code here to make the dir if it does not exist, perhaps that is the cause here. Not entirely sure, but we should be OK to merge the safer code ...

**Review by alvaroaleman — APPROVED** [review]

**Pull request merged**